wowchemy/layouts/partials/functions/parse_social_link.html

```diff
@@ -0,0 +1,23 @@
+{{ $scr := newScratch }}
+
+{{ $pack := or .icon_pack "fas" }}
+{{ $scr.SetInMap "icondict" "icon_pack" $pack }}
```

Comment:
Please use Wowchemy's `snake_case` convention for naming Go variables (including scratch variables) and avoid the use of Hungarian notation (remove postfixed variable types from variable names).

Reply:
Fixed

Reply:
I see you renamed `icondict` to `icon_def`. I'm curious what the meaning of "def" in "icon_def" is?

We named the front matter param as `socials`, so perhaps using the singular `social` or `social_link` fits better here, which also aligns with the function name?

Reply:
It means `definition`. I can rename it to `social_link`, no problem.

Reply:
Done